newindex.html

```diff
@@ -0,0 +1,173 @@
+           .header-page {
+                background-image: url(img/background.jpg);
+                background-repeat: no-repeat;
+                background-size: cover;
+                background-position-x: center;
+                padding-top: 40px;
+                overflow: auto;
+            }
+            .nav-bar {
+            margin-left: auto;
+            display: flex;
+            gap: 20px;
+           }
+           
+           .nav-bar .nav-list {
+                display: flex;
+                gap: 20px;
+            }
+           .action-buttons {
+            margin-left: 40px;
+            display: flex;
+            align-items: flex-end;
+            gap: 15px;
+           }
+           
+           .action-item {
+            background: transparent;
+            border: none;
+           }
+           .icon {
+            width: 16px;
+            height: 16px;
+           }
+           main-content {
+            max-width: 570px;
+           }
+           .container {
+                max-width: 1020px;
+                margin: 0 auto;
+                padding: 0 15px;
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+           }
+           h1, h4 {
+            text-align: center;
+           }
+           .call-to-action {
+            display: flex;
+            justify-content: center;
+            margin-bottom: 120px;
+            gap: 20px;
+           }
+           .quote {
+            padding: 10px 35px;
+            line-height: 26px;
+            border-radius: 37px;
+            border: solid #FFA62B 1px;
+            color: white;
+            background: #FFA62B;
+            cursor: pointer;   
+            font-weight: bolder;
+            transition: 1s;
+        }
+        .quote:hover {
+            transform: scale(1.2); 
+        }
+        @media screen and (min-width: 320px) {
+            .quote {
+                padding: 0;
+                font-size: smaller;
+                padding: 10px 10px;
+            }
+           .learn {
+            padding: 10px 35px;
+            line-height: 26px;
+            border-radius: 37px;
+            border: solid white 1px;
+            color: white;
+            background: transparent;
+            cursor: pointer;
+            font-weight: bolder;
+            transition: 1s;
+        }
+        .learn:hover {
+            transform: scale(1.2); 
+        }
+        @media screen and (min-width: 320px) {
+            .learn {
+                padding: 0;
+                font-size: smaller;
+                padding: 10px 10px;
+                width: 106px;
+            }
+        }
+            @media screen and (min-width: 320px) and (max-width: 768px) {
+            .banner {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                line-height: 30px;
+                margin-bottom: 100px;}
+            }
```

Comment:
HTML checker says that this closing curly brace } at the end is not necessary, but I am not sure if it is a valid point. Maybe you can try locally without it

Reply:
thanks for the comment :) it is because i miss the curly brace on the 112

Reply:
Also, I think your action-buttons are not centered (the search and cart). As I understand this is because they have margin-left:40px in your main code. Try to add them in your media query and set up the margin-left to 0. The rest looks good to me :)

Reply:
thanks for the comment. i tried putting margin left to the whole banner, now i did as you advised, and it worked. Thanks :)